xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -784,10 +824,19 @@
         <Parameter Name="index" Type="System.Int32" Index="0" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1;net-5.0" />
       </Parameters>
       <Docs>
-        <param name="index">To be added.</param>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <param name="index">The zero-based index of the element to get.</param>
+        <summary>Gets the element at the specified index.</summary>
+        <value>The element at the specified index.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This property provides the ability to access a specific element in the collection by using the following syntax: `myCollection[index]`.   
+ The C# language uses the [this](~/docs/csharp/language-reference/keywords/this.md) keyword to define the indexers instead of implementing the <xref:System.Collections.Generic.IList%601.Item%2A> property. Visual Basic implements <xref:System.Collections.Generic.IList%601.Item%2A> as a [default property](~/docs/visual-basic/language-reference/modifiers/default.md), which provides the same indexing functionality.
```

Comment:
```suggestion
 The C# language uses the [this](~/docs/csharp/language-reference/keywords/this.md) keyword to define the indexers instead of implementing the <xref:System.Collections.Generic.IList%601.Item%2A> property. Visual Basic implements <xref:System.Collections.Generic.IList%601.Item%2A> as a [default](~/docs/visual-basic/language-reference/modifiers/default.md) property, which provides the same indexing functionality.
```